knime_extension/src/util/projection.py

```diff
@@ -219,7 +219,7 @@ def pre_processing(
             # TODO: workaround until Java part supports proj strings AP-20328
             from pyproj import CRS  # For CRS Units check
 
-            new_crs = CRS.from_user_input(new_crs).to_wkt()
+            new_crs = CRS.from_user_input(new_crs).to_epsg()
```

Comment:
Are we sure that this always returns an EPSG code? Wouldn't it be saver to check if None was returned and if so use the wkt string as fallback?

Reply:
Yes, as you use UTM-projection based on longitude, it will return a corresponding EPSG code based on the UTM